integration/manifests/executor.yaml

```diff
@@ -24,5 +24,7 @@ spec:
         - /dev/null
         image: kubeapps/integration-tests:latest
         name: integration
-        resources: {}
+        resources:
+          requests:
+            memory: 100Mi
```

Comment:
this avoid `core-dumped` errors when running the browser tests in the executor pod.